src/utils/getErrors.js

```diff
@@ -0,0 +1,27 @@
+const CASES = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+
+function getErrors(originalText, targetCase) {
+  const errors = [];
+
+  if (!originalText) {
+    errors.push({
+      message: `Text to convert is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".`,
```

Comment:
DRY, `Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".` repeated twice. Create variable for error messages